src/main/java/com/commercetools/sync/products/utils/ProductVariantUpdateActionUtils.java

```diff
@@ -318,7 +319,8 @@
             if (newAttribute == null) {
                 final String errorMessage = format(FAILED_TO_BUILD_ATTRIBUTE_UPDATE_ACTION, null,
                     newProductVariant.getKey(), productKey, NULL_PRODUCT_VARIANT_ATTRIBUTE);
-                syncOptions.applyErrorCallback(errorMessage, new BuildUpdateActionException(errorMessage));
+                syncOptions.applyErrorCallback(new SyncException(errorMessage,
+                    new BuildUpdateActionException(errorMessage)), null, null, updateActions);
```

Comment:
I know that current method does not have whole old and new resource objects but maybe we should consider to provide those here? It is also ok to put it into separate issue/PR.

Reply:
It is possible to provide old/new resource. Will add them.

Reply:
Fixed: https://github.com/commercetools/commercetools-sync-java/pull/367/commits/70ca3bafc4e4a6a74e187f87cdcf2c38f3dfecf5